features/before_send.feature

```diff
@@ -0,0 +1,66 @@
+@before_send
+Feature: beforeSend callbacks
+
+Scenario Outline: modifying report via beforeSend in config
+  When I navigate to the URL "/before_send/<type>/a.html"
+  And the test should run in this browser
+  And I let the test page run for up to 10 seconds
+  And I wait for 5 seconds
+  Then I should receive 1 request
```

Comment:
Sounds like a good idea.

I was also considering removing the "And the test should run in this browser" from all the tests I know I don't need that check. Lots of the fixtures harcode "YES" for this purpose, which is pretty redundant.

Reply:
"should run in this browser" seems like a nice general check, though my one concern with it would be accidentally excluding tests. Though given the implementation, that seems unlikely.